playground/infrastructure/helper.py

```diff
@@ -57,10 +62,71 @@ def find_examples(work_dir: str) -> List[Example]:
     Returns:
         List of Examples.
     """
-    examples = [Example()]
+    examples = []
+    for root, _, files in os.walk(work_dir):
+        for filename in files:
+            filepath = os.path.join(root, filename)
+            if match_pattern(filepath):
+                name = get_name(filename)
+                sdk = get_sdk(filename)
+                with open(filepath) as parsed_file:
+                    content = parsed_file.read()
+
+                example = Example(name, sdk, filepath, content)
```

Comment:
Let's extract this part into an internal function like `_get_example`

Reply:
Changed.